packages/vscode-extension/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+// eslint-disable-next-line import/no-commonjs
+module.exports = {
+  // parserOptions: {
+  //   ecmaVersion: 6,
+  //   sourceType: 'module',
+  // },
+  rules: {
+    '@typescript-eslint/naming-convention': 'warn',
```

Comment:
Shouldn't we use the same set of rules for eslint as in package core?

Reply:
This is still the hello-world from https://code.visualstudio.com/api/get-started/your-first-extension.

I kept it here to check if they make sense while developing the extension :)

Reply:
Also, the "core" configs are now in the root. So it's still applying for both packages.